Added VS HTML Hover Support

TanayParikh opened the pull request:
HTML LSP Hover Support now available. I've tested out the functionality by building and running the WTE HTML LSP locally.

Once I got the WTE HTML LSP working, actual integration changes were minimal (leveraging the C# integration).

Keeping as a draft PR until the updates are available in VS (eta Wednesday).

Fixes https://github.com/dotnet/aspnetcore/issues/20963

NTaylorMullen commented:
> Keeping as a draft PR until the updates are available in VS (eta Wednesday).

Any reason we shouldn't just do what we did for Roslyn and have them in earlier?

TanayParikh commented:
> > Keeping as a draft PR until the updates are available in VS (eta Wednesday).
> 
> Any reason we shouldn't just do what we did for Roslyn and have them in earlier?

Sure, we can do that too! 😄

Updated to allow for that, and added note to https://github.com/dotnet/aspnetcore/issues/20962.
